tests/helpers/test_template.py

```diff
@@ -486,26 +487,26 @@ def test_strptime(hass):
         ("2016-10-19", "%Y-%m-%d", None),
         ("2016", "%Y", None),
         ("15:22:05", "%H:%M:%S", None),
-        ("1469119144", "%Y", "1469119144"),
+        ("1469119144", "%Y", 1469119144),
         ("invalid", "%Y", "invalid"),
     ]
 
     for inp, fmt, expected in tests:
         if expected is None:
-            expected = datetime.strptime(inp, fmt)
+            expected = str(datetime.strptime(inp, fmt))
 
         temp = f"{{{{ strptime('{inp}', '{fmt}') }}}}"
 
-        assert template.Template(temp, hass).async_render() == str(expected)
+        assert template.Template(temp, hass).async_render() == expected
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6294088)